shared-ts/jsonTypes.ts

```diff
@@ -0,0 +1,113 @@
+
+enum SourceType {
+    directReport,
+    email,
+    phone,
+    ax,
+    post,
+    feed,
+    radio,
+    tv,
+    web,
+    pager,
+    text,
+    other,
+}
+
+enum Progress {
+    draft, 
+    pendingApproval, 
+    approvedDraft, 
+    open, 
+    closing, 
+    closed, 
+    rejected
+}
+
+enum Reason {
+    Accident,
```

Comment:
There seem to be a few more reasons in the xsd, they also look to start with a lower case letter https://www.siri.org.uk/schema/2.0/xsd/siri_model/siri_situationReasons-v1.1.xsd

Reply:
Added additional reasons and different types of reasons such as environment, personnel and equipment